batches/2019/career-camp-21-october-2019/index.html

```diff
@@ -132,7 +134,24 @@
         </a>
       </div>
     </div>
-    
+    </div>
+    <div class="row">
+    <div class="col">
+
+      <div class="card ml-3 mt-2" style="width: 10rem;">
+        <a href="https://my-resume-68b58.web.app/" target="_blank" class="no-text-decoration">
+          <img src="./students/Dhanush-Kiran/dhanu.jpg" class="card-img-top" alt="profile-pic" style="height: 8.5rem;">
```

Comment:
Remove the style attribute in line number 143.

Reply:
Removed the style attribute

Reply:
Take a fresh pull and then send a PR. Actually we have changed something in the card check that